daprdocs/content/en/operations/hosting/kubernetes/kubernetes-dapr-shared.md

```diff
@@ -0,0 +1,84 @@
+
+Dapr Shared enables two alternative deployment strategies to create Dapr applications using a Kubernetes `Daemonset`for a per-node deployment or a`Deployment` for a per-cluster deployment. 
+
+- **`DaemonSet`:** When specifying to run Dapr as a Kubernetes `DaemonSet` resource, the daprd container runs on each Kubernetes node. This can reduce network hops between the applications and Dapr. 
+- **`Deployment`:** When running Dapr Shared as a Kubernetes `Deployment`, the Kubernetes scheduler decides in which node the Dapr Shared instance will run.
+
+{{% alert title="Dapr Shared deployments" color="primary" %}}
+For each Dapr application, you need to deploy the Dapr Shared chart using different `shared.appId`s.
+{{% /alert %}}
+
+
+
+## Why Dapr Shared?
+
+By default, when Dapr is installed into a Kubernetes cluster, the Dapr control plane injects a Dapr as sidecar to applications annotated with Dapr annotations ( `dapr.io/enabled: "true"`). Sidecars have many advantages including improved resiliency since there is an instance per application and all communication between the application and the sidecar happens without involving the network.
+
+
+<img src="/images/dapr-shared/sidecar.png" width=800 style="padding-bottom:15px;">
+
+While sidecars are Dapr's default deployment strategy, some use cases require other approaches. Let's say you want to decouple the lifecycle of your workloads from the Dapr APIs. A typical example of this is functions, or function-as-a-service runtimes, which might automatically downscale your idle workloads to free up resources. For such cases, keeping the Dapr APIs and all the Dapr async functionalities (such as subscriptions) separate might be required. 
+
+Dapr Shared was created for these scenario, extending the Dapr sidecar model with two new deployment approaches: `DaemonSet` and `Deployment`.
+
+{{% alert title="Important" color="primary" %}}
+No matter which strategy you choose, it is important to understand that in most use cases, you have one instance of Dapr Shared (Helm release) per service (app-id). This means that if you have an application composed of three microservices, each service is recommended to have its own Dapr Shared instance. Try the `hello kubernetes` with Dapr Shared tutorial linked at the bottom of this page. 
+{{% /alert %}}
+
+
+### `DeamonSet` strategy
+
+With Kubernetes `DaemonSet`, you can define workloads that need to be deployed once per node in the cluster. This enables workloads that are running on the same node to communicate with local Dapr APIs, no matter where the Kubernetes `Scheduler` schedules your workload.
+
+<img src="/images/dapr-shared/daemonset.png" width=800 style="padding-bottom:15px;">
+
+{{% alert title="Note" color="primary" %}}
+Since `DaemonSet` installs one instance per node, it consume more overall resources in your cluster when compared to the a`Deployment` for a per-cluster deployment.
```

Comment:
```suggestion
Since `DaemonSet` installs one instance per node, it consumes more overall resources in your cluster, compared to `Deployment` for a per cluster deployment.
```